app/assets/stylesheets/components/_email-link.scss

```diff
@@ -14,6 +18,7 @@
   }
 
   .app-c-email-link__link {
+    @include govuk-font(19);
```

Comment:
Will this affect other email links in the app?

Reply:
it would yes, but this class is only actually used in one place in the whole of finder-frontend ([here](https://github.com/alphagov/finder-frontend/blob/f2f5a582d113868f0e3278b24825e800a53f37ab/app/views/components/_email_link.html.erb#L12)) 

Which is the target of our change.

Reply:
Indeed. My question was where this component then gets used. Looking into it, I can see it's only used by the checker - it's a generic component, so could be used anywhere.